h/services/search_index/_queue.py

```diff
@@ -180,22 +180,32 @@ def sync(self, limit):
 
         LOG.info(dict(counts))
 
-    def _get_jobs_from_queue(self, limit):
-        now = datetime.utcnow()
+    def count(self, tags=None):
+        query = self._job_query
```

Comment:
Good re-use. :+1: We could expand that to include the tags and things as well at some point if there's ever a second caller that wants them.